packages/ibm-products/src/components/CoachmarkStack/CoachmarkStack.js

```diff
@@ -0,0 +1,314 @@
+/**
```

Comment:
There's a few comments/commented lines of code in this file that could either be updated or removed.

Reply:
I believe I got all of them that aren't relevant, worth a double check on me though!